lib/util/itinerary.tsx

```diff
@@ -19,7 +19,7 @@ interface LegWithOtp1HailedCar extends Leg {
   hailedCar?: boolean
 }
 
-interface ItineraryWithOtp1HailedCar extends Itinerary {
+export interface ItineraryWithOtp1HailedCar extends Itinerary {
```

Comment:
Should we rename this since we're using it for non OTP1 trips?

Reply:
I am not sure whether hailed cars were ported to OTP2.

Reply:
They were! But the structure is slightly different I believe